docs/troubleshooting.md

```diff
@@ -1,18 +1,19 @@
+| Cloud Storage FUSE installation fails with an error at build time.                                                                                                                                   | Only specific OS distributions are currently supported, learn more about [Installing Cloud Storage FUSE](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/installing.md).                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| Cloud Storage FUSE not mounting after reboot when entry is present in ```/etc/fstab``` with 1 or 2 as fsck order                                                                                     | Pass [_netdev option](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/mounting.md#persisting-a-mount) in fstab entry (reference issue [here](https://github.com/GoogleCloudPlatform/gcsfuse/issues/1043)). With this option, mount will be attempted on reboot only when network is connected.                                                                                                                                                                                                                                                                                                         |
+| Cloud Storage FUSE gets stuck while doing file operation in high concurrency.                                                                                                                        | This happens when client holds the connection while doing file read/write operation. You might try using `--http-client-timeout=<value>`, this will break the connection if that is hold by client application for too long, and `--max-conns-per-host=<concurrency>`, this will increase the parallel connection.                                                                                                                                                                                                                                                                                                     |
```

Comment:
Cloud Storage FUSE gets stuck while doing file operation in high concurrency -> Cloud Storage FUSE gets stuck when using it to concurrently work with a large number of opened files (reference issue [here](https://github.com/GoogleCloudPlatform/gcsfuse/issues/1040))

This happens when client holds the connection while doing file read/write operation -> This happens when gcsfuse is mounted with http1 client (default) and the application using gcsfuse tries to keep more than value of ```--max-conns-per-host``` number of files opened.  

You might try using `--http-client-timeout=<value>`, this will break the connection if that is hold by client application for too long, and `--max-conns-per-host=<concurrency>`, this will increase the parallel connection. -> You can try (a) Passing  a value higher than the number of files you want to keep open to ```--max-conns-per-host``` flag. (b) Adding some timeout for http client connections using ```--http-client-timeout``` flag.

Reply:
Thanks!